apiserver/provisioner/provisioner.go

```diff
@@ -532,8 +532,9 @@ func (p *ProvisionerAPI) machineVolumeParams(m *state.Machine) ([]storage.Volume
 			return nil, errors.Errorf("cannot get parameters for volume %q", dev.Name())
 		}
 		var options map[string]interface{}
+		var providerType storage.ProviderType
 		if params.Pool != "" {
```

Comment:
I think we guarantee that Pool != "" now right? We should be able to drop this test.